web-ui/src/backup_account/backup_email/backup_email.spec.js

```diff
@@ -118,8 +118,31 @@ describe('BackupEmail', () => {
       expect(preventDefaultSpy).toHaveBeenCalled();
     });
 
-    it('calls onSubmit from props when success', () => {
+    it('calls onSubmit from props with success', () => {
       expect(mockOnSubmit).toHaveBeenCalledWith('success');
     });
   });
+
+  describe('Submit on error', () => {
```

Comment:
We tried doing this initially but we were getting into almost duplicating the two beforeEach's and almost as good as writing separate tests.  But we've now separated them as much as possible, with separate contexts within the same describe.